packages/dashboard/src/components/app.scss

```diff
@@ -1,3 +1,45 @@
 .dw-dashboard {
     height: 100%;
+
+    & .dc-btn {
```

Comment:
If we're going to be adding more of these overrides, perhaps we can add an overrides folder and include them one by one rather than dumping everything in `app.scss`